### ISSUE TITLE
Add Cosmetic Armor [Mod Request]

### ISSUE THREAD

**Issue by Stick404** [ChloeTax/hexxytest3#33]
Basic Cosmetic Armor (found [here](https://modrinth.com/mod/cosmetic-armor/version/1.6.0)). Tested and works on the pack

**Comment by Stick404** [comment]
Tested with normal Terrasteel armor; so it might be a problem with Phantom Ink

**Comment by Twisted-Code** [comment]
Yeah I don't want another player becoming cognitohazardous like Kirby did the one time, K thanks LOL.
